File: THCS/2022_2023/DakLak_2223/Bai1/BAI1.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    int T;
    cin >> T;  // Nhập số lượng test cases
    vector<long long> results(T);  // Danh sách lưu kết quả

    for (int i = 0; i < T; ++i) {
        long long N;
        cin >> N;  // Nhập số nguyên dương N
        results[i] = N * (N + 1) / 2;  // Tính tổng S(N)
    }

    // Xuất kết quả
    for (long long result : results) {
        cout << result << endl;
    }

    return 0;
}

File: THCS/2022_2023/DakLak_2223/Bai1/BAI1.py
def main():
    T = int(input())  # Nhập số lượng test
    results = []  # Danh sách lưu kết quả

    for _ in range(T):
        N = int(input())  # Nhập số nguyên dương N
        S_N = N * (N + 1) // 2  # Tính tổng S(N)
        results.append(S_N)  # Thêm kết quả vào danh sách

    # Xuất ra tất cả kết quả
    print('\n'.join(map(str, results)))

if __name__ == "__main__":
    main()
